.github/workflows/test.yml

```diff
@@ -225,10 +225,9 @@ jobs:
           key: ${{ matrix.os }}-node-v${{ matrix.node }}-nuxt-${{ github.sha }}
 
       - name: restore fixtures
-        uses: actions/cache@v2
+        uses: actions/download-artifact@v2
         with:
-          path: test/fixtures
-          key: ${{ matrix.os }}-node-v${{ matrix.node }}-fixtures-${{ github.sha }}
+          name: fixtures-${{ matrix.os }}-node-v${{ matrix.node }}
```

Comment:
I think we are missing path here :)

Reply:
yes :-)

Reply:
but it does confirm the other bug - I think likely cache wasn't ready to be downloaded and technically there is no contract that it will be in GHA

Reply:
Indeed and cache download issue was frequently happening before too... I hope it works good now :D